master/reference/advanced/etcd-rbac/calico-etcdv2-paths.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Calico key path prefixes in etcd v2
+---
+
+The Paths listed here are the path prefixes that a particular calico component
+needs access to in etcd to function successfully.
+
+## calico/node
+
+XXX Does this change if it is only being used for with docker (libnetwork)?
+
+XXX Does this change any if calico/node is only being used for kubernetes?
+
+| Path                       | Access | Notes |
+|----------------------------|--------|-------|
+| /calico/v1                 |   RW   | Writes out default config, host IP config, libnetwork-plugin writes out WEPs and default policy    |
+| /calico/felix/v1           |   RW   | Felix writes out stats     |
+| /calico/v1/ipam            |   RW   | Creates default IP pools     |
+| /calico/ipam/v2            |   RW   | Allocates IP address for tunl0     |
+| /calico/bgp/v1             |   RW   | Creates default BGP config, and per-node BGP config     |
+
+## felix as a stand alone process
+
+| Path                       | Access | Notes |
+|----------------------------|--------|-------|
+| /calico/v1                 |   R    |       |
+| /calico/felix/v1           |   RW   | Felix writes out stats    |
+
+## confd for bird
```

Comment:
We don't run confd separately though - this is always part of calico/node - so I don't think should be separate.